extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -56,6 +57,49 @@ amp-sidebar[side][i-amphtml-sidebar-opened] {
   animation: none;
 }
 
+amp-sidebar [amp-sidebar-submenu-group],
+amp-sidebar [amp-sidebar-submenu] {
+  position: absolute;
+  outline: none;
+  top: 0;
+  width: 100%;
+  height: 100vh;
+  transform: translateX(0);
+  transition: transform 233ms;
```

Comment:
Any particular reason why `233ms` was chosen?

Reply:
I made the duration to be the same as that of sidebar open/close animations, though I imagine this will be one of the things we should allow developers to customize as I start working on better animation support.